signature-aggregator/aggregator/aggregator.go

```diff
@@ -557,8 +560,9 @@ func (s *SignatureAggregator) isValidSignatureResponse(
 		)
 		return blsSignatureBuf{}, false
 	}
-
-	return signature, true
+	blsSig := blsSignatureBuf{}
+	copy(blsSig[:], signature[:])
+	return blsSig, true
```

Comment:
We can just `return [bls.SignatureLength]byte(signature)` here

Reply:
Addressed in #463